package.json

```diff
@@ -57,7 +57,10 @@
   },
   "devDependencies": {
     "@hmcts/eslint-config": "^1.4.0",
-    "browserify": "^16.2.2",
```

Comment:
Guessing this needs to come back now that you are using `browserify`?

Reply:
indded